test/dotnet-publish.Tests/PublishAppWithBuildDependency.cs

```diff
@@ -17,8 +17,6 @@ public void PublishExcludesBuildDependencies()
             var testInstance = TestAssetsManager.CreateTestInstance("AppWithDirectDependencyAndTypeBuild")
                 .WithLockFiles();
 
-            new RestoreCommand() { WorkingDirectory = testInstance.TestRoot }.Execute().Should().Pass();
```

Comment:
Other tests have this comment:

```
            // Restore the project again in the destination to resolve projects
            // Since the lock file has project relative paths in it, those will be broken
            // unless we re-restore
```

https://github.com/dotnet/cli/blob/f8631fa4b731d4c903dbe8b0b5e5332eee40ecae/test/dotnet-test.Tests/GivenThatWeWantToRunTestsForMultipleTFMsInTheConsole.cs#L33-L35